tests/kafkatest/services/console_consumer.py

```diff
@@ -60,8 +60,9 @@ class ConsoleConsumer(KafkaPathResolverMixin, JmxMixin, BackgroundThreadService)
     def __init__(self, context, num_nodes, kafka, topic, group_id="test-consumer-group", new_consumer=True,
                  message_validator=None, from_beginning=True, consumer_timeout_ms=None, version=DEV_BRANCH,
                  client_id="console-consumer", print_key=False, jmx_object_names=None, jmx_attributes=None,
-                 enable_systest_events=False, stop_timeout_sec=30, print_timestamp=False,
-                 isolation_level="read_uncommitted", jaas_override_variables=None):
+                 enable_systest_events=False, stop_timeout_sec=35, print_timestamp=False,
+                 isolation_level="read_uncommitted", jaas_override_variables=None,
+                 kafka_opts_override="", consumer_properties={}, client_prop_file_override=""):
```

Comment:
consumer_properties is still in parameters
```suggestion
                 kafka_opts_override="", client_prop_file_override=""):
```